### DISCUSSION TITLE
Keeps changing from Ultimate to Standard GPU mode

### DISCUSSION THREAD

**Discussion by april83c** [seerge/g-helper#1440] [Q&A]
Hi, I just switched from Armoury Crate to G-Helper and it's great, but I noticed that sometimes when I boot, it switches from Ultimate GPU mode to Standard GPU mode. Is there a way to make it not do this/always stay in Ultimate?

**Comment by seerge** [comment]
@april83c hello. I don't know what kind of laptop model do you have. But unless you change mode from Ultimate to something else afterwards - it should stay like that after reboot.

GPU mode is saved in BIOS. If you have "Fast boot" enabled in bios, try to disable it (or enable if it's disabled) - mb that causes BIOS to "forget" last state.
